src/main/kotlin/no/nav/dagpenger/journalføring/ferdigstill/JournalføringFerdigstill.kt

```diff
@@ -34,26 +38,27 @@ class JournalføringFerdigstill(val env: Environment, private val oppslagHttpCli
         val inngåendeJournalposter = builder.consumeTopic(INNGÅENDE_JOURNALPOST, env.schemaRegistryUrl)
 
         inngåendeJournalposter
-                .peek { key, value -> LOGGER.info("Processing ${value.javaClass} with key $key") }
-                .filter { _, behov -> behov.getJournalpost().getGsaksakId() != null }
-                .filter { _, behov -> filterJournalpostTypes(behov.getJournalpost().getJournalpostType()) }
-                .foreach { _, value -> ferdigstillJournalføring(value) }
+            .peek { key, value -> LOGGER.info("Processing ${value.javaClass} with key $key") }
+            .filter { _, behov -> behov.hasFagsakId() }
```

Comment:
We should combine the filters, and test the filter-method. We created a shouldBeProcessed method in the other services

Reply:
Fixed- added shouldBeProcessed and test for that